Jenkinsfile

```diff
@@ -1,2 +1,2 @@
 // Build the plugin using https://github.com/jenkins-infra/pipeline-library
-buildPlugin()
+buildPlugin(jenkinsVersions: [null, '2.107.1'])
```

Comment:
> Loading trusted files from base branch master at 670eb0c62db49dd8921489b8fbec6e74c9c0f969 rather than 777bf3062fad1b205228a1818711ea5779748cd1

I manually ran a full build against 2.107.1 and it passed, so probably good enough.